src/main/resources/db/migration/core/V1.00__CLIENT.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE client
```

Comment:
I would prefer the file name V1.0.00, V1.0.01 ... V1.0.99. Meaning version 99 from release 1.0.

Reply:
Done